staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types_jsonschema.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package v1
 
-// JSONSchemaProps is a JSON-Schema following Specification Draft 4 (http://json-schema.org/).
```

Comment:
thanks for the PR... also change it in staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types_jsonschema.go and staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

also, the verify failure is legitimate, the openapi files need regenerating with this change